client/src/Movies/SavedList.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
+import { NavLink } from 'react-router-dom';
```

Comment:
Again, it will be preferable to use Link rather than NavLink if not using it for a navigation